tests/General/MainTest.php

```diff
@@ -7,7 +7,7 @@ public function testRouteResolving() {
         // Normal redirect
         $this->assertEquals('http://tum.sexy/hunger', $router->getTargetOfSub('hunger', null));
         // Subsubdomain redirect to moodle
-        $this->assertContains('https://www.moodle.tum.de/course/view.php?id=', $router->getTargetOfSub('anal', 'm'));
+        $this->assertContains('https://www.moodle.tum.de/course/view.php?id=36704', $router->getTargetOfSub('anal', 'm'));
```

Comment:
I guess the ID will change next semester, so this test will break then. But I guess that's fine, we just need to update the tests next semester together with the ID.

Reply:
No, i don't think so. As long as the don't change there URL structure we should be fine. In Moodle you can see the the sites of previous lectures as well, and the have the same Url structure. So the test should be fine.

Reply:
But it will break when somebody changes the id 🙈